client-course-schedulizer/src/utilities/services/conflictsService.ts

```diff
@@ -55,7 +57,9 @@ export const findConflicts = (schedule: Schedule): Schedule => {
         range1.overlaps(range2) &&
         meeting1.term === meeting2.term &&
         meeting1.days.some(meeting2IncludesDay) &&
-        (meeting1.instructors.some(meeting2IncludesInstructor) || meeting1.room === meeting2.room)
+        (meeting1.instructors.some(meeting2IncludesInstructor) ||
+          meeting1.room === meeting2.room) &&
+        meeting1.sectionName !== meeting2.sectionName
```

Comment:
I think this is good because in theory, this should be correct 99.9% of the time, but I just wanted to point out as a note that `getSectionName()` just uses the first prefix, so there is a chance that it mistakenly thinks 2 sections are the same (but this would require a bad schedule where there was something silly like `MATH/PHIL-100-A` and `MATH/CS-100-A`). Again, this is just a note and probably doesn't require a change.

Reply:
Ah yeah that is true. I didn't think about that. Good catch!